.github/stale.yml

```diff
@@ -7,11 +7,18 @@ exemptLabels:
   - pinned
   - security
 # Label to use when marking an issue as stale
-staleLabel: wontfix
+staleLabel: stale
 # Comment to post when marking an issue as stale. Set to `false` to disable
 markComment: >
-  This issue has been automatically marked as stale because it has not had
-  recent activity. It will be closed if no further activity occurs. Thank you
-  for your contributions.
+  This issue has been automatically marked `stale` because it has not had
+  any activity in the last 60 days. If no further activity occurs within 7
+  days, it will be closed. *Closed does not mean "never"*, just that it has
+  no momentum to get accomplished any time soon.
+
+  See [CONTRIBUTING.md](CONTRIBUTING.md) for more info.
```

Comment:
I'm not really sure these relative links will work in comments, but worth a shot